src/app/user-sign-up/login/login.component.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../layout/_color.scss';
+
+/* Log in Form container */
+.form-container {
+    border-radius: 15px;
+    box-shadow: $offBlack 0px 5px 15px;
+    padding: 20px;
+    width: 400px;
+
+    /* Form group contains input, label tags, validation */
+    .form-group {
+        display: flex;
+        flex-direction: column;
+    
+        /* Email and password input */
+        input[type=password], [type=email] {
+            width: 100%;
+            padding: 12px 20px;
+            margin: 5px 0;
+            display: inline-block;
+            border: 1px solid $lightGrey1;
+            border-radius: 4px;
+            box-sizing: border-box;
+        }
+
+        // Validation for input tag styling
+        input.ng-invalid.ng-touched {
+            border: 1px solid $red;
+        }
+
+        /* Validation message */
+        small {
+            color: $red;
+        }
+    }
+    
+    /*  Submit button*/
+    input[type=submit] {
+        width:100%;
+        background-color: $green;
+        color: $white;
+        padding: 14px 20px;
+        margin: 8px 0;
+        border: none;
+        border-radius: 4px;
+        cursor: pointer;
+    }
+    
+    /* Submit button hover */
+    input[type=submit]:hover {
```

Comment:
check how the SCSS can be written. this can be added in above style
input[type=submit] {
  &:hover {
   }
}